### PR TITLE
Implement autofix for E713 and E714

### PR DESCRIPTION
**Pull request by harupy**
#722

### CODE REVIEW

File: src/pycodestyle/plugins.rs [hunk 1]
```diff
@@ -201,3 +201,62 @@ pub fn literal_comparisons(
...
+) {
+    if matches!(op, Unaryop::Not) {
+        if let ExprKind::Compare {
+            left,
+            ops,
+            comparators,
+            ..
+        } = &operand.node
+        {
+            let should_fix = ops.len() == 1;
```
Comment (harupy) [inline src/pycodestyle/plugins.rs:222]: Should we fix `not (x in y in z)` (= `not (x in y and y in z)`) to `x not in z or y not in z`?

Comment (charliermarsh) [reply src/pycodestyle/plugins.rs:222]: I think it's fine to omit those. They're less in the spirit of the rule.

Comment (charliermarsh) [reply src/pycodestyle/plugins.rs:222]: It looks like Flake8 doesn't even flag those as errors.

Comment (harupy) [reply src/pycodestyle/plugins.rs:222]:
```
> cat resources/test/fixtures/a.py
if not a in b in c:
    pass
> flake8 resources/test/fixtures/a.py --select E713
resources/test/fixtures/E713.py:1:4: E713 test for membership should be 'not in'
```

Comment (charliermarsh) [reply src/pycodestyle/plugins.rs:222]: Haha :)

**Pull request merged**